sql/memo/coster.go

```diff
@@ -386,6 +388,10 @@ func (c *coster) costEmptyTable(_ *sql.Context, _ *EmptyTable, _ sql.StatsReader
 	return 0, nil
 }
 
+func (c *coster) costUnion(_ *sql.Context, _ *Union, _ sql.StatsReader) (float64, error) {
```

Comment:
Seems like we're missing a constant factor of 2 in here

Reply:
2 * ? = ?